kubernetes/cmsweb/services/crabserver.yaml

```diff
@@ -58,6 +58,16 @@ spec:
       #- image: sbelforte/crabserver:3.3.1911.rc3
       - image: cmssw/crabserver #imagetag
         name: crabserver
+        livenessProbe:
+          exec:
+            command:
+            - cmsweb-ping
+            - "--url=http://localhost:8270/crabserver/prod/info"
```

Comment:
we should be careful here since in testbed I think we need /crabserver/preprod/info while in production it would be /crabserver/prod/info. Again we need template in place, but so far we may use substitute in deploy.sh script.